editoast/src/views/paced_train.rs

```diff
@@ -19,17 +28,14 @@ use utoipa::ToSchema;
 
 use super::path::pathfinding::PathfindingResult;
 use super::projection::ProjectPathForm;
+use super::train_schedule::simulation_response;
+use super::train_schedule::train_simulation_batch;
```

Comment:
Module tree is definitely wrong... (nothing to do in this PR, just commenting ^^)